pglookout/pglookout.py

```diff
@@ -290,6 +293,34 @@ def emit_stats(self, state):
         if replication_time_lag is not None:
             self.stats.gauge("pg.replication_lag", replication_time_lag)
 
+    def is_master_observer_new_enough(self, observer_state):
+        if not self.replication_lag_over_warning_limit:
+            return True
+        if not self.current_master or self.current_master not in self.config.get("observers", {}):
+            msg = "Replication lag is over warning limit, but"
+            msg += " current master(%s) is not configured to be polled via observers"
```

Comment:
This is quite suboptimal. Should just use `\` for line continuation, parenthesis, or pass the string directly to `self.log.warning()` so that parenthesis already exist. Also the lines a bit below have been unnecessarily split even though they're below the 120 characters that seems to be the max line length in this repository. Also missing whitespace before `(%s)` in the format string above.